README.md

```diff
@@ -182,10 +180,10 @@ $promises = array(
     $client->imageInspect('ubuntu'),
 );
 
-$inspections = Block\awaitAll($promises, Loop::get());
+$inspections = \React\Async\await(\React\Async\parallel($promises));
```

Comment:
I suggest to go a similar way as @clue did in [reactphp/http#464](https://github.com/reactphp/http/pull/464/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R406) (I assume in my suggestion that this example contains a use statement `use function React\Async\await;`)
```php
$inspections = await(all($promises));
```